src/types.d.ts

```diff
@@ -18,10 +17,9 @@ export interface ComponentProtoBase<Id extends string = string> {
 export interface LanguageProto<Id extends string = string> extends ComponentProtoBase<Id> {
 	grammar: Grammar | ((options: GrammarOptions) => Grammar);
```

Comment:
```suggestion
	grammar: Grammar | ((options?: GrammarOptions) => Grammar);
```

Otherwise, we can't do `grammar.call(this)` without passing mandatory parameters.